ambry-network/src/main/java/com/github/ambry/network/http2/Http2ClientResponseHandler.java

```diff
@@ -66,16 +67,8 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpResponse msg) {
   @Override
   public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
     http2ClientMetrics.http2StreamExceptionCount.inc();
-    logger.info("Exception Caught from inbound.", cause.getMessage());
-    releaseAndCloseStreamChannel(ctx);
-  }
-
-  private void releaseAndCloseStreamChannel(ChannelHandlerContext ctx) {
-    logger.debug("Stream channel is being closed. Stream: {}, Parent: {}", ctx.channel(), ctx.channel().parent());
-    ctx.channel()
-        .parent()
-        .attr(Http2MultiplexedChannelPool.HTTP2_MULTIPLEXED_CHANNEL_POOL)
-        .get()
-        .release(ctx.channel());
+    logger.info("Exception caught from in Http2ClientResponseHandler {} {}. Closing stream channel. Cause: ",
+        ctx.channel().hashCode(), ctx.channel(), cause);
```

Comment:
why include the hashcode in this log message?

Reply:
All stream channels use the same parent channel + H2 stream id as the output string, however, sometimes, H2 stream id is -1. If we want to track this kind of stream, we need a unique id, so I used hashCode().